lib/model/QubitDigitalObject.php

```diff
@@ -1768,7 +1769,30 @@ public function getChildByUsageId($usageId)
     $criteria->add(QubitDigitalObject::PARENT_ID, $this->id);
     $criteria->add(QubitDigitalObject::USAGE_ID, $usageId);
 
-    $result = QubitDigitalObject::getOne($criteria);
+    if ($usageId == QubitTerm::SUBTITLES_ID)
+    {
+      $fileExists = true;
+      $result = array();
+      $i = 0;
+
+      while ($fileExists == true)
+      {
+        $subtitle = QubitDigitalObject::get($criteria)->__get($i, array('defaultValue' => null));
+        if ($subtitle)
+        {
+          $result[] = $subtitle;
+          ++$i;
+        }
+        else
+        {
+          $fileExists = false;
+        }
+      }
+    }
```

Comment:
`QubitDigitalObject::get($criteria)` instead of `getOne` should be enough to get all the related subtitles. I'm not sure if "no results" is returned as `null`, empty `array` or `false`, but you could check that instead of using the while loop.

Reply:
You're right. This works fine, and it's much simpler.

```suggestion
    if ($usageId == QubitTerm::SUBTITLES_ID)
    {
      $result = QubitDigitalObject::get($criteria);
        else
        {
          $fileExists = false;
        }
      }
```